Code/UnitTests/FoundationTest/SimdMath/SimdFloatTest.cpp

```diff
@@ -29,7 +29,7 @@ EZ_CREATE_SIMPLE_TEST(SimdMath, SimdFloat)
     EZ_TEST_BOOL(vInit1F == 2.0f);
 
     // Make sure all components are set to the same value
-#if EZ_SIMD_IMPLEMENTATION == EZ_SIMD_IMPLEMENTATION_SSE
+#if (EZ_SIMD_IMPLEMENTATION == EZ_SIMD_IMPLEMENTATION_SSE) && EZ_ENABLED(EZ_COMPILER_MSVC)
```

Comment:
Why is this stuff MSVC specific?
@C-Core

Reply:
Only MSVC defines __m128 as a union. On clang and gcc __m128 is a opaque data type with no members. Thats why all these tests only work on msvc.